website/docs/Custom-Rules.md

```diff
@@ -20,7 +20,7 @@ rule "MyCustomRule"
 
   buildmessage 'Compiling %(Filename) with MyCustomCC'
   buildcommands 'MyCustomCC.exe -c "%(FullPath)" -o "%(IntDir)/%(Filename).obj"'
-  buildoutputs '%(IntDir)/%(Filename).obj"'
+  buildoutputs '%(IntDir)/%(Filename).obj'
```

Comment:
Should we insert a quote before intdir rather than removing the trailing?

Reply:
I ruled that out :- https://github.com/premake/premake-core/discussions/1935